test/built-ins/Temporal/PlainDateTime/prototype/toZonedDateTime/basic.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.tozoneddatetime
+description: Straightforward case of using an IANA timezone
+features: [Temporal]
+---*/
+
+const dt = new Temporal.PlainDateTime(2020, 1, 1, 0, 0);
+const zdt = dt.toZonedDateTime("UTC");
+
+assert.sameValue(zdt.epochNanoseconds, 1577836800000000000n, "works");
```

Comment:
Here's one place where I think it'd be good to test the time zone.

Reply:
Ah, I see that intuition now -- done.